enterprise/server/remote_execution/blockio/blockio.go

```diff
@@ -523,6 +551,16 @@ func (s *COWStore) copyChunkIfNotDirty(chunkStartOffset int64) (err error) {
 		src.Close()
 	}()
 
+	// note: the src chunk might be smaller than the dst chunk if we resized
+	// and now we're copying the last chunk, since resizing is done lazily.
+	size, err := src.SizeBytes()
+	if err != nil {
+		return err
+	}
+	if size > dstSize {
+		return status.InternalErrorf("chunk source size %d is greater than dest size %d; this is a bug", size, dstSize)
```

Comment:
lol after reading the word `size` so many times it's starting to look really weird to me